### ISSUE TITLE
Windows and Windows.Sample Projects won't open in VS 2017

### ISSUE THREAD

**Issue by kealist** [unosquare/ffmediaelement#353]
# Windows and Windows.Sample Projects won't open in VS 2017

I have installed Framework 4.7.2 and the latest Netcore3 SDKs but I am not able to open the projects due to changes for netcore3.   I assume the project files are tied to a specific version of Netcore prerelease, but I am not sure what that is.

I am able to compile with `dotnet build`:

```
C:\Users\jshireman\Documents\git\ffmediaelement>dotnet build
Microsoft (R) Build Engine version 16.0.443+g5775d0d6bb for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  Persisting no-op dg to C:\Users\jshireman\Documents\git\ffmediaelement\Unosquare.FFME.Windows\obj\Unosquare.FFME.Windows.csproj.nuget.dgspec.json
  Persisting no-op dg to C:\Users\jshireman\Documents\git\ffmediaelement\Unosquare.FFME.Windows.Sample\obj\Unosquare.FFME.Windows.Sample.csproj.nuget.dgspec.json
  Persisting no-op dg to C:\Users\jshireman\Documents\git\ffmediaelement\Unosquare.FFME.Common\obj\Unosquare.FFME.Common.csproj.nuget.dgspec.json
  Restore completed in 47.93 ms for C:\Users\jshireman\Documents\git\ffmediaelement\Unosquare.FFME.Windows.Sample\Unosquare.FFME.Windows.Sample.csproj.
  Restore completed in 47.93 ms for C:\Users\jshireman\Documents\git\ffmediaelement\Unosquare.FFME.Windows\Unosquare.FFME.Windows.csproj.
  Restore completed in 47.93 ms for C:\Users\jshireman\Documents\git\ffmediaelement\Unosquare.FFME.Common\Unosquare.FFME.Common.csproj.
C:\Program Files\dotnet\sdk\3.0.100-preview3-010431\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.RuntimeIdentifierInference.targets(151,5): message NETSDK1057: You are using a preview version of .NET Core. See: https://aka.ms/dotnet-core-preview [C:\Users\jshireman\Documents\git\ffmediaelement\Unosquare.FFME.Common\Unosquare.FFME.Common.csproj]
C:\Program Files\dotnet\sdk\3.0.100-preview3-010431\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.RuntimeIdentifierInference.targets(151,5): message NETSDK1057: You are using a preview version of .NET Core. See: https://aka.ms/dotnet-core-preview [C:\Users\jshireman\Documents\git\ffmediaelement\Unosquare.FFME.Windows.Sample\Unosquare.FFME.Windows.Sample.csproj]
C:\Program Files\dotnet\sdk\3.0.100-preview3-010431\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.RuntimeIdentifierInference.targets(151,5): message NETSDK1057: You are using a preview version of .NET Core. See: https://aka.ms/dotnet-core-preview [C:\Users\jshireman\Documents\git\ffmediaelement\Unosquare.FFME.Common\Unosquare.FFME.Common.csproj]
C:\Program Files\dotnet\sdk\3.0.100-preview3-010431\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.RuntimeIdentifierInference.targets(151,5): message NETSDK1057: You are using a preview version of .NET Core. See: https://aka.ms/dotnet-core-preview [C:\Users\jshireman\Documents\git\ffmediaelement\Unosquare.FFME.Windows\Unosquare.FFME.Windows.csproj]
C:\Program Files\dotnet\sdk\3.0.100-preview3-010431\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.RuntimeIdentifierInference.targets(151,5): message NETSDK1057: You are using a preview version of .NET Core. See: https://aka.ms/dotnet-core-preview [C:\Users\jshireman\Documents\git\ffmediaelement\Unosquare.FFME.Windows.Sample\Unosquare.FFME.Windows.Sample.csproj]
C:\Program Files\dotnet\sdk\3.0.100-preview3-010431\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.RuntimeIdentifierInference.targets(151,5): message NETSDK1057: You are using a preview version of .NET Core. See: https://aka.ms/dotnet-core-preview [C:\Users\jshireman\Documents\git\ffmediaelement\Unosquare.FFME.Windows.Sample\Unosquare.FFME.Windows.Sample.csproj]
C:\Program Files\dotnet\sdk\3.0.100-preview3-010431\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.RuntimeIdentifierInference.targets(151,5): message NETSDK1057: You are using a preview version of .NET Core. See: https://aka.ms/dotnet-core-preview [C:\Users\jshireman\Documents\git\ffmediaelement\Unosquare.FFME.Windows\Unosquare.FFME.Windows.csproj]
  Unosquare.FFME.Common -> C:\Users\jshireman\Documents\git\ffmediaelement\Unosquare.FFME.Common\bin\Debug\netcoreapp3.0\ffme.common.dll
  Unosquare.FFME.Common -> C:\Users\jshireman\Documents\git\ffmediaelement\Unosquare.FFME.Common\bin\Debug\net461\ffme.common.dll
  Unosquare.FFME.Windows -> C:\Users\jshireman\Documents\git\ffmediaelement\Unosquare.FFME.Windows\bin\Debug\net461\ffme.win.dll
  Unosquare.FFME.Windows -> C:\Users\jshireman\Documents\git\ffmediaelement\Unosquare.FFME.Windows\bin\Debug\netcoreapp3.0\ffme.win.dll
  Unosquare.FFME.Windows.Sample -> C:\Users\jshireman\Documents\git\ffmediaelement\Unosquare.FFME.Windows.Sample\bin\Debug\net461\ffmeplay.win.exe
  Unosquare.FFME.Windows.Sample -> C:\Users\jshireman\Documents\git\ffmediaelement\Unosquare.FFME.Windows.Sample\bin\Debug\net472\ffmeplay.win.exe
  Unosquare.FFME.Windows.Sample -> C:\Users\jshireman\Documents\git\ffmediaelement\Unosquare.FFME.Windows.Sample\bin\Debug\netcoreapp3.0\ffmeplay.win.dll

Build succeeded.
    0 Warning(s)
    0 Error(s)

Time Elapsed 00:00:11.25
```
Just not open the projects in VS 2017

## Issue Categories

 - [x] Bug
 - [ ] Feature Request
 - [ ] Question
 - [ ] Not sure

**Comment by mariodivece** [comment]
The project files have been moved to visual studio 2019 preview. Vs 2019 will be released on April 2 but you can download the preview version now.

**Comment by kealist** [comment]
Please update your Readme accordingly

**Comment by mariodivece** [comment]
http://brianlagunas.com/enable-net-core-3-in-visual-studio-2017/

**Comment by kealist** [comment]
Unfortunately, I had tried following that before I made this ticket.  There have been too many "versions" of breaking changes over the preview period with WPF and netcore3, that it is likely only going to work with whatever set of changes are for the current version.  

   I was able to get VS 2019 installed at work, but the project will not compile due to the following error:

```
Your project file doesn't list 'win' as a "RuntimeIdentifier". You should add 'win' to the "RuntimeIdentifiers" property in your project file and then re-run NuGet restore.	Unosquare.FFME.Windows.Sample			
```

Of course no amount of trying to add the various windows RuntimeIdentifier to the project seems to resolve that issue for me.    No restores seem to help VS compile (it will still compile from command line though, but won't help with debugging easily)
